gpslogger/src/main/res/xml/pref_general.xml

```diff
@@ -34,9 +34,7 @@
         android:key="hide_notification_from_status_bar"
         android:defaultValue="false"
         />
-    <Preference
-        android:title="@string/gpslogger_permissions_rationale_title"
-        android:key="permissions_required" />
```

Comment:
I've moved the codebase to AndroidX.  
The existing permission workflow made use of the Material Dialogs library. 
But the version I'm using isn't compatible with AndroidX.
The later versions are compatible with AndroidX but aren't compatible with Java. 
So I've had to remove the ability to go through the permission workflow from the settings screen.
I can't convert this Preference to the AndroidX version without ripping out Material Dialogs.